test/test_other.py

```diff
@@ -12087,8 +12087,8 @@ def test_missing_malloc_export(self):
       }
     ''')
     self.do_runf('unincluded_malloc.c', (
-      "malloc() called but not included in the build - add '_malloc' to EXPORTED_FUNCTIONS",
-      "free() called but not included in the build - add '_free' to EXPORTED_FUNCTIONS"))
+      'malloc() called but not included in the build - add `_malloc` to EXPORTED_FUNCTIONS',
+      'free() called but not included in the build - add `_free` to EXPORTED_FUNCTIONS'), assert_all=True)
```

Comment:
Looks like `assert_all` was needed all along, IIUC?

Reply:
Yes, I think the author (probably me!) assumed that it was the default.